Misc/NEWS.d/next/IDLE/2017-12-25-18-48-50.bpo-32411.vNwDhe.rst

```diff
@@ -0,0 +1,2 @@
+In browser.py, remove extraneous sorting by line number since dictionary was
```

Comment:
`browser` is not a top-level module, I think this entry should mention `idlelib.browser` or `the IDLE xyz plugin`.